# Get Depth Image

---

sensor_msgs/Image color_image
sensor_msgs/Image depth_image
